README.md

```diff
@@ -1,19 +1,38 @@
-[![OpenCollective](https://opencollective.com/sinon/sponsors/badge.svg)](#sponsors) ![npm downloads per month](https://img.shields.io/npm/dm/sinon.svg)
-
-Standalone and test framework agnostic JavaScript test spies, stubs and mocks (pronounced "sigh-non", named after [Sinon, the warrior](https://en.wikipedia.org/wiki/Sinon)).
-
-[![Sauce Test Status](https://saucelabs.com/browser-matrix/sinonjs.svg)](https://saucelabs.com/u/sinonjs)
+<h1 align=center>
+    <a href="http://sinonjs.org" title="Sinon.JS">
+        <img alt="Sinon.JS" src="http://sinonjs.org/assets/images/logo.png">
+    </a>
+    <br>
+    Sinon.JS
+</h1>
+
+<p align=center>
+    Standalone and test framework agnostic JavaScript test spies, stubs and mocks (pronounced "sigh-non", named after <a href="https://en.wikipedia.org/wiki/Sinon">Sinon, the warrior</a>).
+</p>
+
+<p align=center>
+<a href="https://www.npmjs.com/package/sinon"><img src="https://img.shields.io/npm/v/sinon.svg?style=flat" alt="npm version"></a>
+<a href="https://gitter.im/sinonjs/sinon?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join the chat at https://gitter.im/sinonjs/sinon"></a>
+<a href="http://travis-ci.org/sinonjs/sinon"><img src="https://secure.travis-ci.org/sinonjs/sinon.svg?branch=master" alt="Build status"></a>
+<a href="https://www.bithound.io/github/sinonjs/sinon"><img src="https://www.bithound.io/github/sinonjs/sinon/badges/score.svg" alt="bitHound Score"></a>
+<a href="https://saucelabs.com/u/sinonjs"><img src="https://saucelabs.com/buildstatus/sinonjs" alt="Sauce Test Status"</a>
+<a href="https://coveralls.io/github/sinonjs/sinon"><img src="https://coveralls.io/repos/github/sinonjs/sinon/badge.svg" alt="Coverage Status"></a>
+<a href="#backers"><img src="https://opencollective.com/sinon/backers/badge.svg" alt="OpenCollective"></a>
+<a href="#sponsors"><img src="https://opencollective.com/sinon/sponsors/badge.svg" alt="OpenCollective"></a>
+<a href="https://www.npmjs.com/package/sinon" target="_blank"><img src="https://img.shields.io/npm/dm/sinon.svg" alt="npm downloads per month"></a></p>
+</p>
```

Comment:
One `</p>` too many, isn't there? At least that explains why the markdown diff gets italicized from here on (to signify syntax error). Shouldn't matter much, though.